mapie/tests/test_classification.py

```diff
@@ -779,6 +992,38 @@ def test_y_is_list_of_string(
     assert y_ps.shape == (X.shape[0], len(np.unique(y)), n_alpha)
 
 
+def test_same_results_prefit_split() -> None:
```

Comment:
Could you just iterate over strategies ?

Reply:
Over methods like "raps", "score", "cumulated_score" ?

Reply:
Done.